include/miopen/miopen.h

```diff
@@ -2820,11 +2820,14 @@ typedef enum
  * @param fusePlanDesc  Pointer to a fusion plan (output)
  * @param fuseDirection Horizontal or Vertical fusion (input)
  * @param inputDesc     Descriptor to tensor for the input (input)
+ * @param fusion_mode   id to represent fusion plan's mode. eg: for conv + bias + activ would be 0
+ * (miopen_fusion_cba)
  * @return              miopenStatus_t
  */
 MIOPEN_EXPORT miopenStatus_t miopenCreateFusionPlan(miopenFusionPlanDescriptor_t* fusePlanDesc,
                                                     const miopenFusionDirection_t fuseDirection,
-                                                    const miopenTensorDescriptor_t inputDesc);
+                                                    const miopenTensorDescriptor_t inputDesc,
+                                                    int fusion_mode);
```

Comment:
This changes the API, I am not sure why this is necessary ?

Reply:
These contant/enums can be used internally in MIOpen to check for specific case/ logging etc. However, such an enum in the public facing API defeats its purpose for composability.